### PR TITLE
utils_netperf: Add params to support latest netperf version

### PR DESCRIPTION
**Pull request by yanglei-rh**
Add params to support latest netperf version.
ID:1833873
Signed-off-by: Lei Yang <leiyang@redhat.com>

### CODE REVIEW

**Comment by yanglei-rh** [comment]
@PaulYuuu Please help me review it,thanks.

File: virttest/utils_netperf.py [hunk 1]
```diff
@@ -124,8 +124,13 @@ def pack_compile(self, compile_option=""):
         build_type = {"aarch64": "aarch64-unknown-linux-gnu"}
         build_arch = self.session.cmd_output("arch", timeout=60).strip()
         np_build = build_type.get(build_arch, build_arch).strip()
-        setup_cmd = "./configure --build=%s %s > /dev/null 2>&1" % (np_build,
-                                                                    compile_option)
+        if self.netperf_dir == "/var/tmp/netperf-2.7.1":
```
Comment (PaulYuuu) [inline virttest/utils_netperf.py:127]: Check the absolute path is not a good idea, better to use RegExp to judge its version from `self.netperf_file` and then add `autogen.sh` before currently command if need.

**Comment by PaulYuuu** [comment]
@vivianQizhu Also request for your review, thanks.

File: virttest/utils_netperf.py (hunk 1 above)
Comment (PaulYuuu) [reply virttest/utils_netperf.py:127]: Checking if `autogen.sh` exists is an easier method.

Comment (yanglei-rh) [reply virttest/utils_netperf.py]: I have updated it,please help me review it again.Thanks.

File: virttest/utils_netperf.py [hunk 2]
```diff
@@ -124,8 +124,9 @@ def pack_compile(self, compile_option=""):
         build_type = {"aarch64": "aarch64-unknown-linux-gnu"}
         build_arch = self.session.cmd_output("arch", timeout=60).strip()
         np_build = build_type.get(build_arch, build_arch).strip()
-        setup_cmd = "./configure --build=%s %s > /dev/null 2>&1" % (np_build,
-                                                                    compile_option)
+        setup_cmd = "./autogen.sh > /dev/null 2>&1 &&" \
+                    " ./configure --build=%s %s > /dev/null 2>&1" % (np_build,
+                                                                     compile_option)
```
Comment (vivianQizhu) [inline virttest/utils_netperf.py:129]: I not sure about your change, so both versions have `autogen.sh` script? You will fail the setup if it does not exist?

Comment (yanglei-rh) [reply virttest/utils_netperf.py:129]: Yes,  The existing netperf version will release the 'autogen.sh' script after decompression.

**Comment by vivianQizhu** [comment]
@PaulYuuu Request for your ack, thanks.

**Pull request merged**